src/js/common/components/AlertBox.jsx

```diff
@@ -45,8 +45,8 @@ class AlertBox extends React.Component {
     let closeButton;
     if (this.props.onCloseAlert) {
       closeButton = (
-        <button className="va-alert-close" onClick={this.props.onCloseAlert}>
-          <i className="fa fa-close"></i>
+        <button className="va-alert-close" aria-label="Close notification" onClick={this.props.onCloseAlert}>
```

Comment:
There are several places in vets.gov where we create our own "va-alert-close" buttons instead of using this component for some reason (maybe that should change), but took this label from one of them: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/disability-benefits/components/Notification.jsx

Oddly, the 508 checks don't complain about this button even though it's missing an aria-label:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/components/Disclaimer.jsx#L28
so I left it alone.